manifest_staging/charts/aad-pod-identity/values.yaml

```diff
@@ -9,6 +9,10 @@ image:
   repository: mcr.microsoft.com/k8s/aad-pod-identity
   imagePullPolicy: Always
 
+# One or more secrets to be used when pulling images
+imagePullSecrets: []
```

Comment:
I think we should change this to use 

```
# imagePullSecrets:
#   - name: myRegistryKeySecretName
```

instead of just the secrets names as list to conform with other values variables in the chart